java/registry/src/main/java/io/opensaber/registry/service/impl/RegistryServiceImpl.java

```diff
@@ -166,21 +166,21 @@ public void deleteEntityById(Shard shard, String uuid) throws Exception {
      * @return
      * @throws Exception
      */
-    public String addEntity(Shard shard, String jsonString, String userId) throws Exception {
+    public String addEntity(Shard shard, JsonNode inputJson, String userId) throws Exception {
```

Comment:
I asked to retain rootNode as the var name to minimize changes.

Reply:
renamed for minimal change effect.